practice-two/src/stores/reducers.js

```diff
@@ -0,0 +1,35 @@
+import { ACTION_TYPES } from '@constants';
+
+export const initialCustomer = {
+  customers: [],
+};
+
+export const customerReducer = (state, action) => {
+  const { type, payload } = action;
+  const { customers } = state;
+  switch (type) {
+    case ACTION_TYPES.LIST: {
```

Comment:
it is action type so should use verb

Reply:
I changed LIST to GET_LIST
![image](https://github.com/datdragon777/React-Training/assets/74806826/95da6340-dfc5-4ebf-b63f-636846947b98)